styles/main.scss

```diff
@@ -1481,3 +1481,170 @@ blockquote p:last-child {
 .posts-list-block:hover .post-img:after {
   height: 100%;
 }
+
+/*audio-player ==============================================*/
+
+.audio {
+  position: relative;
+  width: 100%;
+  height: 53px;
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  padding: 10px 16px 7px 10px;
+  background: #f2f5f7;
+  border-radius: 4px;
+  margin-bottom: 18px;
+}
+
+.audio button {
+  border: none;
+  cursor: pointer;
+  padding: 0;
+}
+
+.audio button .pause-icon {
+  object-fit: cover;
+}
+
+.audio .play {
+  position: relative;
+  width: 36px !important;
+  height: 33px;
+  background: #ffffff;
+  box-shadow: 0px 4px 16px rgba(0, 0, 0, 0.12);
+  border-radius: 50%;
+  flex: 1 0 33px;
+}
+
+.audio .play img {
+  position: absolute;
+  left: 54%;
+  top: 50%;
+  transform: translate(-50%, -50%);
+}
+
+.audio .volume {
+  background-color: transparent;
+  margin-bottom: 2px;
+}
+
+.audio .volume-bar {
+  opacity: 0;
+  pointer-events: none;
+
+  position: absolute;
+  display: flex;
+  width: 157px;
+  height: 38px;
+  padding: 13px;
+
+  transform: rotate(-90deg);
+  bottom: 120px;
+  right: -60px;
```

Comment:
two magic numbers to position correctly volumeBar component, do you have any ideas or thought about this?